tdvt/tdvt/tabquery.py

```diff
@@ -3,6 +3,7 @@
 
 from .resources import *
 tab_cli_exe = ''
+logical_rewrite_appoption = ''
```

Comment:
I think it would be simpler if you took this out, then you wouldn't need the else clause on line 81 either.